ds-cr/syllabus.md

```diff
@@ -1,7 +1,28 @@
-Syllabus 
-- Introduction to version control with Git 
-- Collaboration with Git and Github 
-- Testing 
-- Modular Code Development 
-- Continuous Integration 
-- Code Documentation 
+Introduction to version control with Git 
+* Tracking changes: `git add` & `git commit`
+* Exploring history, checking out older versions
+* Ignoring things with .gitignore files
+* Github remotes
+
+Collaboration with Git and Github
+* Creating pull requests
+* Review process
+* Good practices for collaboration
+* Contributing to repositories with forks
+
+Code Documentation 
+* In-code documentation
+* Readme files
+* Writing documentation with sphinx and ReadTheDocs
```

Comment:
In practice we do this with Github pages instead of RTD, but both are in the lesson material. Might be nice to mention GH pages here?

Reply:
I added GH pages.